.github/workflows/ontology_update.yml

```diff
@@ -0,0 +1,37 @@
+name: Updated Ontologies
+
+on:
+  pull_request:
+    paths:
+      - '**/cellxgene_schema_cli/cellxgene_schema/ontology_files/owl_info.yml'
+
+jobs:
+  gene-processing:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.8
+      - name: Python cache
+        uses: actions/cache@v1
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements*.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+      - name: ontology-processing
+        run: |
+          pip install -r cellxgene_schema_cli/requirements.txt
+          make download-ontologies -C cellxgene_schema_cli
+      - name: Commit
+        run: |
+          git config user.name github-actions
+          git config user.email github-actions@github.com
+          git add ./cellxgene_schema_cli/cellxgene_schema/ontology_files/all_ontology.json.gz
+          git commit -m "AUTO: update ontologies"
+          git push
```

Comment:
I noticed the push tests don't run after this commit--I'm not sure why though, the configuration of push_tests looks like it should trigger. We do have some tests around the 'validate' command that we should run after an ontology update

Reply:
It only runs when a change is detected in the owl_info.yml. If you push another commit to the same branch that doesn't change that file, then the GHA will not run.

Reply:
github docs https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow